### PR TITLE
More doc details

### PR DESCRIPTION
**Pull request by Gaetanbrl**
Doc PR preview  is available here : 

https://github.com/jdev-org/ansible/blob/doc-master-0621/README.md

* Insert infos to take account  of #79 #78 
* A little more example & command to detail install

### CODE REVIEW

**Comment by landryb** [comment]
i'm fine with the mapstore addition to sec-proxy config, can this be done in a separate PR fixing only #83 ? Generally it's cleaner to have one PR per problem..

as for the geoserver thing, the last packages shipped for the stable branches should have the CORS filter for tomcat uncommented by default, cf georchestra/georchestra#3358, so i dont think it should be needed to document that.

**Comment by Gaetanbrl** [comment]
> Generally it's cleaner to have one PR per problem..

My bad... i change it now.

**Comment by Gaetanbrl** [comment]
> as for the geoserver thing, the last packages shipped for the stable branches should have the CORS filter for tomcat uncommented

It's not the case in actual ansible master branch and i needed to change comment Jetty filter. All filters are uncommented by default.

**Comment by Gaetanbrl** [comment]
mmmmh PR close by error sorry. Create new one.

File: README.md [hunk 1]
```diff
@@ -4,23 +4,60 @@ A simple [ansible](http://docs.ansible.com) playbook to deploy a fullblown [geOr
...
+
+- **Set MapStore2-georchestra config extensions**
+
+To import plugins, Tomcat (or Jetty) will need to write into a plugins folder. He can't by default and you get some errors.
+In fact, this folder could be the default datadir (/etc/georchestra/datadir) but it's not fully recomended.
+> [More details here](http://docs.georchestra.geo-solutions.it/fr/latest/configuration/application/index.html?highlight=extensions#dynamic-files)
+
+So, we suggest you to set an alternative mapstore plugins directory. To do that, open `/etc/default/tomcat-georchestra` and add this JVM option :
+
+`-Dgeorchestra.extensions= /target/path/extensions \`
```
Comment (landryb) [inline README.md:53]: im not sure tomcat will like the extra space after the = :)

File: README.md [hunk 2]
```diff
@@ -4,23 +4,60 @@ A simple [ansible](http://docs.ansible.com) playbook to deploy a fullblown [geOr
...
- * setup variables for your own instance in ```playbooks/georchestra.yml```,
- * point to the IP of a host where you have ssh-with-passphrase root access in the ```hosts``` file (ideally, an lxc container, or a vm, whatever suits you)
+
+ * Clone source :
+
+    `git clone https://github.com/georchestra/ansible.git`
+
+ * setup variables for your own instance in ```playbooks/georchestra.yml```
+
+ * Open `ansible/hosts` file
```
Comment (landryb) [inline README.md:28]: why `ansible` subdir here ? at that point one can assume the admin went to the subdir to edit `playbooks/georchestra.yml`

**Comment by landryb** [comment]
fixed by followup in a5dcd26, thanks !

File: README.md (hunk 2 above)
Comment (Gaetanbrl) [reply README.md:28]: indeed... this was to show where is hosts file (inside ansible root folder).

**Pull request merged**